src/esg_fastapi/api/models.py

```diff
@@ -61,236 +63,1132 @@ class ESGSearchQueryBase(BaseModel):
+        },
+    )
+    project: OptionalParam[MultiValued[str]] = Query(
+        None,
+        title="Project",
+        description="The `project` identifier, e.g., `CMIP5`, `CMIP6`. <sup>1</sup><br><br><small><sup>1</sup>: [ESGF Search "
+        "Documentation](https://esgf-node.llnl.gov/esgf_docs/search_api.html)</small>",
+        openapi_examples={
+            "CMIP5": {"summary": "CMIP5 Project", "description": "Project CMIP5.", "value": "CMIP5"},
+            "CMIP6": {"summary": "CMIP6 Project", "description": "Project CMIP6.", "value": "CMIP6"},
+            "obs4MIPs": {"summary": "obs4MIPs Project", "description": "Project obs4MIPs.", "value": "obs4MIPs"},
+        },
+    )
+    quality_control_flags: OptionalParam[MultiValued[str]] = Query(
+        None,
+        title="Quality Control Flags",
+        description="Flags indicating the `quality control` status of the data, e.g., `passed`.",
+        openapi_examples={"passed": {"summary": "Passed", "description": "Quality control passed.", "value": "passed"}},
+    )
+    range: OptionalParam[MultiValued[str]] = Query(None, title="Range")
+    realm: OptionalParam[MultiValued[str]] = Query(
+        None,
+        title="Realm",
+        description=(
+            "The modeling `realm` (e.g., `atmos`, `ocean`). <sup>1</sup><br><br>"
+            "<small><sup>1</sup>: [CMIP6 Global Attributes, DRS, Filenames, Directory Structure, and CV's]"
+            "(https://wcrp-cmip.github.io/WGCM_Infrastructure_Panel/Papers/CMIP6_global_attributes_filenames_CVs_v6.2.7.pdf), "
+            "Table 1</small>"
+        ),
+        openapi_examples={
+            "atmos": {"summary": "Atmosphere", "description": "Atmosphere realm.", "value": "atmos"},
+            "ocean": {"summary": "Ocean", "description": "Ocean realm.", "value": "ocean"},
+            "land": {"summary": "Land", "description": "Land realm.", "value": "land"},
+        },
+    )
+    realm_drs: OptionalParam[MultiValued[str]] = Query(None, title="Realm DRS")
+    region: OptionalParam[MultiValued[str]] = Query(
+        None,
+        title="Region",
+        description="The geographical `region` covered by the data, e.g., `global`.",
+        openapi_examples={"global": {"summary": "Global", "description": "Global region.", "value": "global"}},
+    )
+    regridding: OptionalParam[MultiValued[str]] = Query(None, title="Regridding")
+    run_category: OptionalParam[MultiValued[str]] = Query(None, title="Run Category")
+    Science_Driver: OptionalParam[MultiValued[str]] = Query(
+        None,
+        alias="Science Driver",
+        title="Science Driver",
+        description="The `science driver` for the experiment, e.g., `aerosol`.",
+        openapi_examples={"aerosol": {"summary": "Aerosol", "description": "Aerosol science driver.", "value": "aerosol"}},
+    )
+    science_driver_: OptionalParam[MultiValued[str]] = Query(None, alias="science driver", title="Science Driver (Alternative)")
```

Comment:
Maybe we should set more aliases? Can they be a list and include `Science_Driver`? We could handle others above in the same way.